.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,25 @@
 <!--
-Note: This checklist is a reminder of our shared engineering expectations. Feel free to change it, although assigning a GitHub reviewer and the items in bold are required.
+Note: This checklist is a reminder of our shared engineering expectations. 
+The items in Bold are required
+If your PR involves UI changes:
+    1. Upload screenshots or screencasts that illustrate the changes before / after
+    2. Add them under the UI changes section (feel free to add more columns if needed)
+    3. Make sure these changes are tested in API 21 and API 26
+If your PR does not involve UI changes, you can remove the **UI changes** section
 -->
 
-Task/Issue URL: 
-Tech Design URL: 
-CC: 
+Task/Issue URL:
 
 **Description**:
 
 
 **Steps to test this PR**:
-1. 
-1. 
 
+***Feature***
```

Comment:
minor suggestion: while we're in here changing the format, can we use proper markdown headings instead of just ** 

e.g. 👇 
`### Description`

Reply:
oh that's actually a great idea, I'll add it. Thanks @CDRussell